Packs/trendMicroDsm/Scripts/script-TrendMicroGetPolicyID.yml

```diff
@@ -33,7 +33,8 @@ type: python
 subtype: python2
 tags:
 - Trend Micro
-comment: Returns a table of policies and their TrendMicro IDs
+deprecated: true
+comment: Deprecated. No available replacement.
```

Comment:
is trendmicro-get-policy an available replacement?

Reply:
not precisely, trendmicro-get-policy returns info about a certain policy according to the policy_id it receives while the script takes a policy name and returns its ID